.jslintrc

```diff
@@ -0,0 +1,37 @@
+{
+	"predef": [
+		"document",
+		"navigator",
+		"console",
+		"window",
+		"django",
+		"gettext",
+		"windowname_to_id",
+		"jQuery",
+		"WebSocket",
```

Comment:
you can remove this

Reply:
`gettext` is used in `preview.js` (line 50 and 71) and `unsaved_changes.js` (line 43), so I've left it there.